tools/myst/enc/enc.c

```diff
@@ -748,7 +748,7 @@ int myst_load_fssig(const char* path, myst_fssig_t* fssig)
 #define ENCLAVE_SECURITY_VERSION 1
 #define ENCLAVE_DEBUG true
 
-#define ENCLAVE_HEAP_SIZE 131072
+#define ENCLAVE_HEAP_SIZE (256 * 1024)
```

Comment:
We have tried very hard to keep the OE heap size small. This would throw away the work for one extreme case. Could this be avoided?

Reply:
I'm working on an alternative now. :)

Reply:
I posted a new fix that flattens the buffer on the kernel side.